app/components/ToggleOption/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+*
+* ToggleOption
+*
+*/
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';  // eslint-disable-line import/no-unresolved
+
+function ToggleOption(props) {
+  return (
+    <option value={props.value}>
+      <FormattedMessage {...props.message} />
```

Comment:
this is throwing an error in browser...
```
warning.js:44 Warning: Only strings and numbers are supported as <option> children.
```
we will need to use `injectIntl` and formatMessage api here to pass string